plugins/out_s3/s3.c

```diff
@@ -991,7 +1005,8 @@ static int upload_data(struct flb_s3 *ctx, struct s3_file *chunk,
     void *payload_buf = NULL;
     size_t payload_size = 0;
     size_t preCompress_size = 0;
-
+    char create_time_string[20];
```

Comment:
why do you have two strings?

Reply:
One for multipart one for put-object. I'm not sure if there is a situation where these two can be together can have two create_time.